### PR TITLE
Fixes for es6 usage

### PR DESCRIPTION
**Pull request by gberaudo**

### CODE REVIEW

File: buildtools/webpack.commons.js [hunk 1]
```diff
@@ -110,6 +150,11 @@ const config = {
       htmlRule,
       iconRule,
       cursorRule,
+      ngeoRule,
+      ngeoExamplesRule,
+      gmfRule,
+      gmfExamplesRule,
+
```
Comment (sbrunner) [inline buildtools/webpack.commons.js]: extra space...

File: package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "ngeo",
+  "name": "ngeo-es6",
```
Comment (sbrunner) [inline package.json]: Oups

File: src/goog.asserts.prod.js_ [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
-exports.assert = function(condition, opt_message, var_args) {};
-exports.assertNumber = function(value, opt_message, var_args) {};
-exports.assertString = function(value, opt_message, var_args) {};
-exports.assertFunction = function(value, opt_message, var_args) {};
-exports.assertObject = function(value, opt_message, var_args) {};
-exports.assertArray = function(value, opt_message, var_args) {};
-exports.assertBoolean = function(value, opt_message, var_args) {};
-exports.assertElement = function(value, opt_message, var_args) {};
-exports.assertInstanceof = function(value, type, opt_message, var_args) {};
+exports.assert = function(condition, opt_message, var_args) { return condition;};
```
Comment (sbrunner) [inline src/goog.asserts.prod.js_]: missing space...

Comment (sbrunner) [reply src/goog.asserts.prod.js_]: eslint want's to be on a separate line...

File: buildtools/create-es6-package [hunk 4]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+sed -i 's/"name": "ngeo",/"name": "ngeo-es6",/g' package.json
+sed -i 'sY/ngeo\\/Y/ngeo-es6\\/Yg' buildtools/webpack.*.js
+sed -i 'sY~openlayersY../../../node_modules/openlayersYg' contribs/gmf/less/base.less
+sed -i 'sY~bootstrapY../../../node_modules/bootstrapYg' contribs/gmf/less/base.less
```
Comment (sbrunner) [inline buildtools/create-es6-package]: Why this?

Comment (gberaudo) [reply buildtools/create-es6-package]: This is a temporary workaround. See https://jira.camptocamp.com/browse/GSEPF-47.

**Comment by gberaudo** [comment]
@sbrunner, the build is now passing. What do you want to do concerning CSS handling?

**Pull request merged**